src/main/java/taxi/dao/ManufacturerDao.java

```diff
@@ -4,7 +4,7 @@
 import java.util.Optional;
 import taxi.model.Manufacturer;
 
-public interface ManufacturerDao {
+public interface ManufacturerDao extends GenericDao<Manufacturer, Long> {
```

Comment:
You can delete all methods under, that you declared in GenericDao. Same in other Dao and Service